### ISSUE TITLE
Is there a simple method can directly use layoutlmv3 and test some image samples?

### ISSUE THREAD

**Issue by 4daJKong** [microsoft/unilm#839]
I only want to input some images without labels to this model and return a result including bbox, labels, etc. However, the code is too complicate for a novice... I tried to  import AutoModel from transformers in huggingface.co but how to use it, I only write some parts of code and I am still stuck in input my images.
```
import numpy as np
from pathlib import Path
from PIL import Image

from transformers import AutoTokenizer, AutoProcessor, AutoModel

tokenizer = AutoTokenizer.from_pretrained("microsoft/layoutlmv3-base")
processor = AutoProcessor.from_pretrained("microsoft/layoutlmv3-base")
model = AutoModel.from_pretrained("microsoft/layoutlmv3-base")

print(model.eval())

image_path = Path("data/images/0001.jpg")
image = Image.open(image_path).convert("RGB")

encoding = processor(image, return_tensors="pt", truncation=True, max_length=512)
encoding = processor(image, return_tensors="pt", return_offsets_mapping=True)
offset_mapping = encoding.pop('offset_mapping')
import torch

device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')

print(encoding.keys())
for k,v in encoding.items():
    encoding[k] = v.to(device)
    #print(k,v.shape)

model.to(device)  
outputs = model(**encoding)

last_hidden_states = outputs.last_hidden_state
print(last_hidden_states)

predictions, labels, metrics = trainer.predict(test_dataset)
predictions = np.argmax(predictions, axis=2)

logits = outputs.logits
print(logits.shape)
predictions = logits.argmax(-1).squeeze().tolist()
print(predictions)
labels = encoding.labels.squeeze().tolist()
print(labels)
```

Which line is necessary and how to get the prediction result from a pretrained model (microsoft/layoutlmv3) ?

**Comment by HYPJUDY** [comment]
The bbox information is a necessary input for layoutlm models. But you can skip the manual input of bbox and extract it automatically with an OCR processor. @NielsRogge provides two examples based on [LayoutLMv2](https://huggingface.co/spaces/nielsr/LayoutLMv2-FUNSD) and [DiT](https://huggingface.co/spaces/nielsr/dit-document-layout-analysis): you just need to input an image to get its bbox and labels ([more details](https://github.com/microsoft/unilm/issues/417)). I hope these examples meet your need and help you modify the pre-trained model to LayoutLMv3.
